api/services/prompt_service.py

```diff
@@ -109,15 +111,3 @@ def fetch_prompt_response(self, prompt):
         split_prompt_by_hyphen = helpers.split_prompt_by_hyphen(prompt)
         split_prompt_by_underscore = helpers.split_prompt_by_underscore(split_prompt_by_hyphen[-1])
         return split_prompt_by_underscore[1] if len(split_prompt_by_underscore) > 1 else None
-
-    def add_user_module_content(self, user_details, content_id):
```

Comment:
We don't need this any more. Created a separate service to handle these changes.